src/components/code_editor/code_editor.test.tsx

```diff
@@ -117,7 +117,7 @@ describe('OuiCodeEditor', () => {
         expect(blurSpy).toHaveBeenCalled();
       });
 
-      test('pressing escape in ace textbox will enable overlay', () => {
+      test.skip('pressing escape in ace textbox will enable overlay', () => {
```

Comment:
We should make a followup issue if we decide to merge this without unskipping these